### PR TITLE
feat: add how to contribute

### LINKED ISSUE #1440: [Site]: Create/Improve "Contribute to Apache APISIX" page
### Current State

**Description of the current state/issue**:

Create a new page called "How to contribute" under Contribute to APISIX section in the first list. Provide links to other pages in the table below.

**Link to the page**:

For Page, content ask @qu
**Screenshots (if relevant)**:

image

### Desired State

**Description of the change you are proposing**:

We need the well-described open-source contributor page for APISIX.

### Would you like to fix this issue?

No. Please assign another contributor

### PR DESCRIPTION
**Pull request by guoqqqi**
Fixes: #1440 

Changes:

Screenshots of the change:

### CODE REVIEW

**Comment by guoqqqi** [comment]
Hello, @Boburmirzo You can see how the page looks via the preview link. At the moment we still need a Google form to collect information from contributors
https://deploy-preview-1486--apache-apisix.netlify.app/docs/general/how-to-contribute/
image

File: website/docs/general/how-to-contribute.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+title: How to Contribute
+keywords:
+  - API gateway
+  - APISIX
+  - Apache APISIX
+  - contributor guide
+description: Guidelines for Apache APISIX contributors.
+---
+
+**• Engage with the Community • Build Connections • Earn Rewards**
```
Comment (SkyeYoung) [inline website/docs/general/how-to-contribute.md:12]:
```suggestion
**• Engage with the Community • Build Connections • Earn Rewards**
```

Should we ignore the lint error?

**Review by SkyeYoung — COMMENTED** [review]
^

File: website/docs/general/how-to-contribute.md (hunk 1 above)
Comment (guoqqqi) [reply website/docs/general/how-to-contribute.md:12]: I think we can ignore it, it's not a formatting error.

**Comment by Boburmirzo** [comment]
> Hello, @Boburmirzo You can see how the page looks via the preview link. At the moment we still need a Google form to collect information from contributors https://deploy-preview-1486--apache-apisix.netlify.app/docs/general/how-to-contribute/ image

@guoqqqi Yes, it looks great. Thanks for your help on that. 

The growth team planned to prepare a Google form to claim the certificate. Let me check with them what's the progress.

**Comment by Boburmirzo** [comment]
> > Hello, @Boburmirzo You can see how the page looks via the preview link. At the moment we still need a Google form to collect information from contributors https://deploy-preview-1486--apache-apisix.netlify.app/docs/general/how-to-contribute/ image
> 
> @guoqqqi Yes, it looks great. Thanks for your help on that.
> 
> The growth team planned to prepare a Google form to claim the certificate. Let me check with them what's the progress.

You can use this link:

https://forms.gle/DhPL96LnJwuaHjHU7

**Review by SkyeYoung — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**